.github/workflows/ubuntu.yml

```
@@ -215,6 +215,14 @@ jobs:
           batch-name: valid-v910
           os: ${{ env.os }}
 
+      - name: Run tests introduced in 9.2
```

Please edit lines 210, 202, etc. with real version name like here (8.7.0, 9.1.0, etc.)
```suggestion
      - name: Run tests introduced in 9.2.0
```